Source/Core/Core/IOS/USB/Host.cpp

```diff
@@ -19,22 +19,26 @@
 #ifdef __LIBUSB__
 #include <libusb.h>
 #endif
+
 #ifdef _WIN32
 #include <SetupAPI.h>
 #include <cfgmgr32.h>
 #include <devpkey.h>
-
-#include "Common/CommonFuncs.h"
 #endif
 
 #include "Common/ChunkFile.h"
+#include "Common/CommonFuncs.h"
```

Comment:
Is this include being used by anything anymore?

Reply:
Doesn't seem like it, I'll remove it.